cmd/mr_unapprove.go

```diff
@@ -23,16 +23,32 @@ var mrUnapproveCmd = &cobra.Command{
 		lab mr unapprove upstream -m "A helpfull\nComment" --force-linebreak`),
 	PersistentPreRun: labPersistentPreRun,
 	Run: func(cmd *cobra.Command, args []string) {
+		var (
+			linebreak = false
+			canUnapprove = false
+		)
+
```

Comment:
Instead of creating this vars at the beginning, create them right before using them, like you did in the "approve" code, so can keep the code standard for both actions.